docs/data-model.md

```diff
@@ -65,14 +65,14 @@ erDiagram
   User {
     uuid id PK
     string email UK
+    enum service "Placements or Claims"
```

Comment:
I'm happy to go this way -- but I wonder if it's useful to use the same pattern we're using above for schools (placements_enabled, claims_enabled). It would be consistent, but also, the unique constraint would then just be on the `email`, rather than `email`, `service` combination. 

Perfectly happy to go with the enum approach. Just putting this out there for a point of discussion.